src/package_details.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>About Chef Workstation</title>
+    <link rel="stylesheet" href="../assets/css/main.css">
+    <link rel="stylesheet" href="../assets/css/about.css">
+  </head>
+  <body>
+    <h1 id="chef-workstation-package-details">Chef Workstation Package Details</h1>
+    <ul>
+    <script>
+      document.write("Querying...")
```

Comment:
This works and is fine. As this evolves we might want to consider separating the js from the html. I also wonder if we shouldn't get the versions from the manifest and display something more formatted. Nothing to change here just thinking out load.